.eslintrc

```diff
@@ -39,7 +39,9 @@
         "**/actions/**/common*.*js",
         "**/actions/common/*",
         "**/sources/common/*",
-        "**/sources/**/common*.*js"
```

Comment:
These changes are unrelated, but I had to fix this to address some false positive errors Husky / ESLint raised in the `pre-commit` hook.